server/plugin.go

```diff
@@ -21,7 +21,7 @@ type Plugin struct {
 }
 
 // ServeHTTP demonstrates a plugin that handles HTTP requests by greeting the world.
-func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
+func (p *Plugin) ServeHTTP(_ *plugin.Context, w http.ResponseWriter, _ *http.Request) {
```

Comment:
Is this a GoLang-CI requirement? I think we should disable this rule, since this reduces the utility of the template if I have to fill-in the variable names. This also doesn't seem to be idiomatic -- the example on https://golang.org/pkg/net/http/#example_ListenAndServe doesn't use an `_` even though the variable is unused.

Reply:
It's required by `unparam`. I personally like the style, but I see that it reduces the utility. Removing it.